docs/changelogs/CHANGELOG-1.3.md

```diff
@@ -32,6 +33,7 @@
 
 ### Updated
 
+- Upgrade Node-Exporter to v1.3.1
```

Comment:
Yeap - exactly as it was stated in Additional remarks chapter in this PR.
I didn't put the task no along with the task name (_Upgrade Node-Exporter to v1.3.1_) because I just wanted to stay aligned with the other elements in this section.

Reply:
Add a task id/link

Reply:
The remarks were taken on board within [this commit](https://github.com/epiphany-platform/epiphany/pull/2807/commits/95e08dbfdcdfe64730425287770d4dbdb8547f22).